core/server/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -168,7 +168,7 @@ public boolean inodeIdExists(long id) {
   public boolean inodePathExists(AlluxioURI path) {
```

Comment:
let's change the input argument name from `path` to `uri` to avoid the awkward `path.getPath()`